<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>521</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>JP Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../git/pyside1/256jpconverter.ico</normaloff>../../git/pyside1/256jpconverter.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#B0BEC5;</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #818489;</string>
   </property>
   <widget class="QPushButton" name="button_convert">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>105</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#1565c0;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input1">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>190</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeeeee;font-size:20px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_from_metric">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>5</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#00600f;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>From 
Metric</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_from_jpmeasure">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>5</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#00600f;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>From JP 
Measures</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_to_jpyear">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>75</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#087f23;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>To JP 
Years</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>290</y>
      <width>105</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;background-color:#515151;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_to_jpmeasure">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>75</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#087f23;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>To JP 
Measures</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_zodiac">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>5</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#00600f;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Zodiac
Years</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_from_jpyear">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>5</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#00600f;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>From JP 
Years</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_instructions1">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>140</y>
      <width>391</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font:arial; font-size:20px; color:#ffffff; padding:11px 0px 11px 0px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="label_output2">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>260</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="conv_layout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>310</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="conv_layout_contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>310</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_to_metric">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>75</y>
      <width>105</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color:#087f23;font:arial;font-size:17px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>To 
Metric</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_header1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>365</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_instructions2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>310</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font:arial; font-size:16px; color:#ffffff; padding:11px 0px 11px 11px; line-height:1.4; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_instructions2</zorder>
   <zorder>conv_layout</zorder>
   <zorder>button_convert</zorder>
   <zorder>button_from_metric</zorder>
   <zorder>button_from_jpmeasure</zorder>
   <zorder>button_to_jpyear</zorder>
   <zorder>button_to_jpmeasure</zorder>
   <zorder>button_zodiac</zorder>
   <zorder>button_from_jpyear</zorder>
   <zorder>label_instructions1</zorder>
   <zorder>input1</zorder>
   <zorder>label_output2</zorder>
   <zorder>button_to_metric</zorder>
   <zorder>label_header1</zorder>
   <zorder>button_exit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;</string>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Measures">
    <property name="title">
     <string>&amp;Measurements</string>
    </property>
    <addaction name="action_from_metric"/>
    <addaction name="action_to_metric"/>
    <addaction name="separator"/>
    <addaction name="action_from_jpmeasure"/>
    <addaction name="action_to_jpmeasure"/>
   </widget>
   <widget class="QMenu" name="menu_Years">
    <property name="title">
     <string>&amp;Calendar</string>
    </property>
    <addaction name="action_from_jpyear"/>
    <addaction name="action_from_jpyear_historic"/>
    <addaction name="action_to_jpyear"/>
    <addaction name="separator"/>
    <addaction name="action_zodiac"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Measures"/>
   <addaction name="menu_Years"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_from_jpyear">
   <property name="text">
    <string>From &amp;Japanese years (modern)</string>
   </property>
  </action>
  <action name="action_to_jpyear">
   <property name="text">
    <string>To Japanese &amp;years</string>
   </property>
  </action>
  <action name="action_zodiac">
   <property name="text">
    <string>&amp;Zodiac years</string>
   </property>
  </action>
  <action name="action_from_metric">
   <property name="text">
    <string>From &amp;Metric</string>
   </property>
  </action>
  <action name="action_to_metric">
   <property name="text">
    <string>To M&amp;etric</string>
   </property>
  </action>
  <action name="action_from_jpmeasure">
   <property name="text">
    <string>&amp;From Japanese measurements</string>
   </property>
  </action>
  <action name="action_to_jpmeasure">
   <property name="text">
    <string>&amp;To Japanese measurements</string>
   </property>
  </action>
  <action name="action_from_jpyear_historic">
   <property name="text">
    <string>From Japanese years (&amp;historic)</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>button_from_metric</tabstop>
  <tabstop>button_from_jpmeasure</tabstop>
  <tabstop>button_from_jpyear</tabstop>
  <tabstop>button_zodiac</tabstop>
  <tabstop>button_to_metric</tabstop>
  <tabstop>button_to_jpmeasure</tabstop>
  <tabstop>button_to_jpyear</tabstop>
  <tabstop>input1</tabstop>
  <tabstop>button_convert</tabstop>
  <tabstop>button_exit</tabstop>
  <tabstop>label_output2</tabstop>
  <tabstop>conv_layout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
